geomesa-spark/geomesa-spark-jts/pom.xml

```diff
@@ -29,9 +29,15 @@
             <version>${spark.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts-io</artifactId>
+            <version>${jts.version}</version>
+        </dependency>
         <dependency>
             <groupId>com.vividsolutions</groupId>
             <artifactId>jts-core</artifactId>
+            <version>${jts.version}</version>
```

Comment:
@elahrvivaz do you think we should be managing the versions in the root pom?  Or did we make a decision to handle this differently since it might be moving out?

Reply:
yeah, we should probably put it in the root pom. looks like the other dependencies are coming from there. `${spark.version}` is only referenced here b/c it's redefining the scope to provided.